sample.env

```diff
@@ -0,0 +1,10 @@
+OPENAPI_KEY="seufhsfewnf-efhewbfbebifoewbfoe"
```

Comment:
I guess it is OPENAI_API_KEY right? Prefer using semantic names for ENV vars in this case?